RIGS/forms.py

```diff
@@ -140,4 +140,42 @@ class Meta:
         fields = ['is_rig', 'name', 'venue', 'start_time', 'end_date', 'start_date',
                   'end_time', 'meet_at', 'access_at', 'description', 'notes', 'mic',
                   'person', 'organisation', 'dry_hire', 'checked_in_by', 'status',
-                  'collector', 'purchase_order']
+                  'collector']
+
+
+class BaseClientEventAuthorisationForm(forms.ModelForm):
+    tos = forms.BooleanField(required=True, label="Terms of hire")
```

Comment:
True, I suspect the reason this happens is
```
test = 'no I don't accept'

if test:
    print("They accepted")
```
But I think it's reasonable to assume clients won't mess with the HTML form, so this is a non-issue